### PR TITLE
Add readme for the Frontend UI

### PR DESCRIPTION
**Pull request by dit7ya**
As the title says. Also fixes `vast rest` -> `vast web`.

### CODE REVIEW

File: plugins/web/README.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
 
 This plugin provides a web server and access to the REST frontend for VAST.
 
+The following describes the backend web server part of the web plugin. For information regarding the frontend vast UI, see [this](./ui/README.md).
```
Comment (dominiklohmann) [inline plugins/web/README.md:5]:
```suggestion
The following describes the backend web server part of the web plugin. For information regarding the frontend VAST UI, see [this](./ui/README.md).
```

In code or slugs, use `vast`, but when talking about the product, use VAST as the productized spelling.

Nit: Please limit this to 80 cols per the style guide.

File: plugins/web/ui/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# For a flake enabled nix setup
+nix shell nixpkgs#nodejs-18_x nixpkgs#yarn
+
+# Otherwise
+nix-shell -p nodejs-18_x yarn
+```
+
+### Develop
+
+This folder can be developed independently from the rest of the VAST package.
```
Comment (dominiklohmann) [inline plugins/web/ui/README.md:19]:
```suggestion
To develop the UI independently from VAST itself:
```

We try to use active voice in documentation.

File: plugins/web/ui/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+1. Install the nodejs dependencies
+
+```fish
+yarn install
+```
+
+2. You need a running vast instance with the web server enabled in the dev mode. For example
+
+```fish
+vast --plugins=bundled start '--commands="web server --mode=dev"'
```
Comment (dominiklohmann) [inline plugins/web/ui/README.md:30]:
```suggestion
vast start '--commands="web server --mode=dev"'
```

We merged a change that causes VAST to load all plugins by default recently.

File: plugins/web/ui/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+This folder can be developed independently from the rest of the VAST package.
+
+1. Install the nodejs dependencies
+
+```fish
+yarn install
+```
+
+2. You need a running vast instance with the web server enabled in the dev mode. For example
```
Comment (dominiklohmann) [inline plugins/web/ui/README.md:27]:
```suggestion
2. You need a running VAST instance with the web server enabled in the dev mode. For example
```

File: plugins/web/ui/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+```fish
+vast --plugins=bundled start '--commands="web server --mode=dev"'
+```
+
+3. (optional) If your vast server's REST API endpoint is different from the default (`http://localhost:42001/api/v0`), copy the `.env.example` to `.env` and edit that.
```
Comment (dominiklohmann) [inline plugins/web/ui/README.md:33]:
```suggestion
3. (optional) If your VAST server's REST API endpoint is different from the default (`http://localhost:42001/api/v0`), copy the `.env.example` to `.env` and edit that.
```

**Review by dominiklohmann — APPROVED** [review]
Thanks for taking the initiative on this, much appreciated!

Over time, I think some of this should also be reflected on vast.io, maybe in the develop or contribute sections. But for now this is definitely a really good improvement.

**Review by Dakostu — APPROVED** [review]
Approved (after Dominik's comments)

**Pull request merged**